src/main/webapp/script.js

```diff
@@ -0,0 +1,162 @@
+const MONTHS = ['January', 'February', 'March', 'April', 'May', 'June', 'July',
+                'August', 'September', 'October', 'November', 'December'];
+const OCD_STATE_TITLE = 'state:';
+const OCD_DISTRICT_TITLE = 'district:';
+const STATE_CODE_LENGTH = 2;
+const ELECTION_LIST_TITLE = 'Elections coming up in ';
+const COUNTRY_NAME = ' the United States';
+const NO_ELECTIONS_TEXT = 'No elections coming up soon.';
+const LEARN_MORE_BUTTON_TEXT = 'Learn more';
+const INVALID_DATE_TEXT = 'Invalid date';
+
+/**
+ * Fetch the list of upcoming elections and display relevant ones according
+ * to the state selected by the user.
+ */
+function listElections() {
```

Comment:
Can you add a todo to write tests?

https://medium.com/@rishabhsrao/mocking-and-testing-fetch-with-jest-c4d670e2e167
This may help for how to mock the fetch() calls.

Reply:
Sure, I'm planning to do tests tomorrow. Thanks for the link!